File: 1 тиждень/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>
float normfloat(string lol);
int main(void)
{
    int count=0; 
    float cash = normfloat("Change: ");
    int coins = round(cash*100);
    if(coins>=25)
    {
        int k = coins/25;
        count+=k;
        int m = coins%25;
        if(m==0)
        {
            printf("%i\n", count);
            return 0;
        }
        coins=m;
    }
    if(coins>=10)
    {
      int k = coins/10;
        count+=k;
        int m = coins%10;
        if(m==0)
        {
            printf("%i\n", count);
            return 0;
        }
        coins=m;  
    }
    if(coins>=5)
    {
      int k = coins/5;
        count+=k;
        int m = coins%5;
        if(m==0)
        {
            printf("%i\n", count);
            return 0;
        }
        coins=m;  
    }
    count+=coins;
    printf("%i\n", count);
    return 0;
}
float normfloat(string lol)
{
    float k;
    do 
    {
        k = get_float("%s", lol);
    }
    while(k<0);
    return k;
}

File: 6 тиждень/cash/cash.py
from cs50 import get_float
coins = 0
while True:
    zdacha = get_float("Change owed: ")
    if zdacha >= 0:
        break
k = round(zdacha*100)
if k >= 25:
    n = k//25
    coins += n
    k %= 25
if k >= 10:
    n = k//10
    coins += n
    k %= 10
if k >= 5:
    n = k//5
    coins += n
    k %= 5
coins += k
print(coins)